globus_sdk/config.py

```diff
@@ -143,6 +143,17 @@ def get_service_url(environment, service):
     return url
 
 
+def get_http_timeout(environment):
+    p = _get_parser()
+    value = p.get("http_timeout", environment=environment,
+                  failover_to_general=True, check_env=True,
+                  type_cast=int)
```

Comment:
Not a huge deal but if we have `http_timeout` documented as a `float` maybe we should be casting to that here instead of `int`?

Reply:
Good catch. Will fix.